### PR TITLE
HSC-226: Fix concepts for Examen obstétrical form that break Orders

### PR DESCRIPTION
**Pull request by kazlaw**
Link for ticket here -->https://mekomsolutions.atlassian.net/browse/HSC-226

[Screencast from 30-04-24 10:10:39.webm](https://github.com/CRUDEM/openmrs-config-hsc/assets/36139599/ba7e6ea0-305d-45cd-b2c8-7db1d32cecb0)

### CODE REVIEW

File: configuration/bahmniforms/obstetrical_examination_1.json [hunk 1]
```diff
@@ -7,57 +7,83 @@
         "published": true,
         "auditInfo": {
             "creator": {
-                "uuid": "0e13890d-da59-11ee-85e8-0242ac140008",
-                "display": "admin",
+                "uuid": "A4F30A1B-5EB9-11DF-A648-37A07F9C90FB",
+                "display": "daemon",
```
Comment (Ruhanga) [inline configuration/bahmniforms/obstetrical_examination_1.json]: Let's leave such fields untouched. You only want to copy over the `"resources":` section with the necessary translations, not forgetting to update the relevant reference UUIDs.

**Review by Ruhanga — CHANGES_REQUESTED** [review]

File: configuration/bahmniforms/obstetrical_examination_1.json (hunk 1 above)
Comment (kazlaw) [reply configuration/bahmniforms/obstetrical_examination_1.json]: This has been handled [here](https://github.com/CRUDEM/openmrs-config-hsc/pull/74/commits/2712245a8e29bdc4c82a10a103954ed4549fa154)

Comment (kazlaw) [reply configuration/bahmniforms/obstetrical_examination_1.json]: [Screencast from 07-05-24 12:08:05.webm](https://github.com/CRUDEM/openmrs-config-hsc/assets/36139599/59b86f41-4ba5-415a-8ad2-f4da6a207492)

**Comment by kazlaw** [comment]
Hi @Ruhanga please have a look when you get a chance

**Review by Ruhanga — COMMENTED** [review]
@kazlaw, after these changes, where you able to confirm that initializer loads the configs without errors?

**Comment by kazlaw** [comment]
Yes @Ruhanga all was okay

**Review by Ruhanga — APPROVED** [review]

**Pull request merged**